.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+Before submitting a pull request, please read
```

Comment:
Things that are instructions to the PR Creator can be made in <!-- comment blocks -->

Reply:
I think most of your comment got cut-off